<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>179</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>13</y>
     <width>321</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLabel" name="MergeThreshold_LB">
      <property name="text">
       <string>Merge Threshold (dalton)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="MinInt_LB">
      <property name="text">
       <string>Min Intensity (%)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDoubleSpinBox" name="MinInt_SBox">
      <property name="decimals">
       <number>5</number>
      </property>
      <property name="minimum">
       <double>0.000010000000000</double>
      </property>
      <property name="maximum">
       <double>0.900000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.000010000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDoubleSpinBox" name="IntRatio_SBox">
      <property name="decimals">
       <number>4</number>
      </property>
      <property name="minimum">
       <double>0.000100000000000</double>
      </property>
      <property name="maximum">
       <double>10.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.001000000000000</double>
      </property>
      <property name="value">
       <double>0.001000000000000</double>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDoubleSpinBox" name="MergeThreshold_SBox">
      <property name="decimals">
       <number>4</number>
      </property>
      <property name="minimum">
       <double>0.000100000000000</double>
      </property>
      <property name="maximum">
       <double>10.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.001000000000000</double>
      </property>
      <property name="value">
       <double>0.000500000000000</double>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="IntRatio_LB">
      <property name="text">
       <string>Intensity Ration (%)</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="SetOptions_btn">
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
